src/core/default/commands/dynamodb/shell.py

```diff
@@ -0,0 +1,132 @@
+import cmd
+import json
+import csv
+from io import StringIO
+from argparse import ArgumentParser
+from typing import List, Tuple
+
+import aurora_data_api
+from rich.console import Console
+from rich.table import Table
+
+from core.constructs.commands import BaseCommand
+from core.default.commands.dynamodb.utils import (
+    get_dynamodb_info_from_cdev_name,
+    dynamodb_item_action,
+)
+
+import core.default.mappers.aws_client as aws_client
+import boto3
+import json
+
+
+class shell(BaseCommand):
+    help = """
+        Open an interactive shell to a non-relational db.
+    """
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        parser.add_argument(
+            "resource",
+            type=str,
+            help="The database to execute on. Name must include component name. ex: comp1.myDb",
+        )
+        parser.add_argument(
+            "--put_item",
+            help="put item command, format: field:field_name , field_type: string or int , value: "
+            "value_field; field:field2_name, field_type: string, value:value_field2 (...)",
+        )
+        parser.add_argument(
+            "--delete_item",
+            type=str,
+            help="delete item command, format: field:field_name , field_type: string or int , value: "
+            "value_field; field:field2_name, field_type: string, value:value_field2 (...)",
+        )
+        parser.add_argument(
+            "--get_item",
+            type=str,
+            help="get item command, format: field:field_name , field_type: "
+            "string or int , value: "
+            "value_field; field:field2_name, field_type: string, value:value_field2 (...)",
+        )
+        parser.add_argument(
+            "--put_item_from_json",
+            type=str,
+            help="put item command from a json file in single quotes",
```

Comment:
Can you add something to the help message to let them know what the json should look like. You can include a link to the dynamodb documentation if that helps.